lib/ts/recipe/oauth2client/api/authorisationUrl.ts

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { send200Response } from "../../../utils";
+import STError from "../../../error";
+import { APIInterface, APIOptions } from "../";
+import { UserContext } from "../../../types";
+
+export default async function authorisationUrlAPI(
+    apiImplementation: APIInterface,
+    _tenantId: string,
+    options: APIOptions,
+    userContext: UserContext
+): Promise<boolean> {
+    if (apiImplementation.authorisationUrlGET === undefined) {
+        return false;
+    }
+
+    const redirectURIOnProviderDashboard = options.req.getKeyValueFromQuery("redirectURIOnProviderDashboard");
```

Comment:
Not sure if the name makes sense, but it should be OK. Please add a TODO note to check this name later.

Reply:
done